torch/_inductor/decomposition.py

```diff
@@ -476,6 +476,15 @@ def add(
     y_is_complex_tensor = torch.is_tensor(y) and y.is_complex()
     if not x_is_complex_tensor or not y_is_complex_tensor:
         return NotImplemented
+
+    if x.ndim == 0 and y.ndim == 0:
```

Comment:
if I understand correctly, that would force you to default path, and 0d + 0d would not be compiled